src/or/networkstatus.c

```diff
@@ -1528,7 +1528,7 @@ networkstatus_consensus_has_ipv6(const or_options_t* options)
     return
        cons->consensus_method >= MIN_METHOD_FOR_A_LINES_IN_MICRODESC_CONSENSUS;
   } else {
-    return cons->consensus_method >= MIN_METHOD_FOR_A_LINES;
+    return 1;
```

Comment:
I think it's ok for us to modify this client code. As of 1 May 2018, we will only support 0.2.9, which creates consensuses with method 25 and later.

And this is edge-case code. It is only useful:
* on networks with consensus methods below 13 (no IPv6 addresses in the full consensus),
* on clients using full descriptors (IPv6 addresses in descriptors),
* with IPv6-supporting relays,
* and IPv6-supporting fallbacks,
* with an IPv6-only client.